pkg/controller/useraccount/useraccount_controller.go

```diff
@@ -317,3 +407,22 @@ func newIdentity(userAcc *toolchainv1alpha1.UserAccount, user *userv1.User) *use
 func ToIdentityName(userID string) string {
 	return fmt.Sprintf("%s:%s", config.GetIdP(), userID)
 }
+
+func containsString(items []string, value string) bool {
```

Comment:
Perhaps these utility funcs should have a better home?

Reply:
yes ;)